Azure/sonic-buildimage
# [device][accton]: Update for AS5835-54T

* Update Accton-AS5835-54T to adapt xxx.config.bcm based on the latest update of Device-Specific File Directory Structure.
* Update Accton-AS5835-54T upon the port link / led test / PING out / SCP upon self-bringing up the new SAI version, 3.5.2.3.

Signed-off-by: polly_hsu@edge-core.com

**- What I did**
Update Accton-AS5835-54T upon the port link / led test / PING out / SCP upon self-bringing up the new SAI version, 3.5.2.3

**- How I did it**
1. Update Accton-AS5835-54T to adapt xxx.config.bcm based on the latest update of Device-Specific File Directory Structure the SONiC Porting Guide.
2. Update Accton-AS5835-54T against the latest update of the SONiC Porting Guide (https://github.com/Azure/SONiC/wiki/Porting-Guide).

**- How to verify it**
1. Target-verified the port mapping of 48x10G+6x100G as expected.
2. Target-verified the port link of 48x10G+6x100G as expected.
3. Target-verified the port led of 48x10G+6x100G as expected, however, we off-loaded the MV2 LED BINs to our self-brought up SAI libraries due to SONiC NOT supporting them currently.
3. We off-loaded the MV2 SDK config setting to our self-brought up SAI libraries, and will update it upon more verification check.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
